### ISSUE TITLE
Formatting for in-line code should have background?

### ISSUE THREAD

**Issue by gdevenyi** [swcarpentry/styles#76]
I think perhaps the in-line code segments would stand-out more if they had a background, like the classic github style.

Like `here`

**Comment by gvwilson** [comment]
They would stand out more, but a lot of people find that distracting.  
Take it to a vote of the maintainers?

**Comment by gdevenyi** [comment]
Yeah, reasonable.

**Comment by fmichonneau** [comment]
+1

**Comment by shwina** [comment]
-1 (somewhat reluctantly). I like in-line code background, but in portions with "dense" inline code, it can become highly distracting.

**Comment by iglpdc** [comment]
+1 (but agreeing with @shwina on large portions of inline). I find the current version a bit hard to read, because you "feel" a difference, but sorta cannot say what's going on.

Maybe we could limit the use of inline code when very short (filenames, program and library names) and move the rest (any statement or command) to its own paragraph.

**Comment by gdevenyi** [comment]
Agree, inline code is for short snippets, if large chunks exist that's a lesson formatting issue that should be fixed.

**Comment by shwina** [comment]
What I meant was paragraphs like these:

Et `utinam theophrastus` has, `legimus` instructior pro `ut`. `Qui te` nobis prompta, `id` `quo` `quod quodsi` fierent. No fuisset `adolescens` sea, vocent `aeterno aperiam` id mel, quo no `labore` audire officiis. Vix et purto verterem vituperata, ex elit mentitum `urbanitas vis`.

Maybe it's just me that finds these disorientating :-)

**Comment by gvwilson** [comment]
@k8hertweck can you please poll maintainers 'n' point them at this issue to see if they have opinions?

**Comment by daattali** [comment]
+1 I came here to suggest this issue.

I do understand that sometimes it may look ugly, but it makes it much clearer what is code and what is text/punctuation. I just went over the entire unix lesson, and there were many times where I wasn't sure where in a paragraph the inline code ends, and if the punctuation is part of the command or not.

One example: [here](http://swcarpentry.github.io/shell-novice/04-pipefilter/)

> run the command `wc *.pdb`. wc is the

Without the background It's very easy to think that the period and second `wc` are also part of the command
